### DISCUSSION TITLE
Decrypting previously downloaded audiobooks

### DISCUSSION THREAD

**Discussion by TobiMMXIX** [audiamus/BookLibConnect#83] [Q&A] [answered]
German:
Besteht die Möglichkeit zuvor heruntergeladene Hörbücher zu entschlüsseln? Ich habe viele Hörbücher über die Windows 10 App von Audible oder auch direkt über die Webseite heruntergeladen. Diese Hörbücher kann ich aktuell nicht mehr konvertieren und müsste sie mit diesem Tool erneut herunterladen.

English:
Is it possible to decrypt previously downloaded audio books? I have downloaded many audiobooks via Audible's Windows 10 app or directly from the website. I am currently unable to convert these audiobooks and would have to download them again using this tool.

**Comment by audiamus** [comment, marked as answer]
Re-download may not be needed.

If you still have the computer on which you'd installed the official Audible app for Win 10 and if the app was activated for the account you downloaded the books in question, you will have the necessary "activation bytes" stored on that computer. Install [AAX Audio Converter](https://github.com/audiamus/AaxAudioConverter) - the sibling of Book Lib Connect - on the same computer and process your books.

**Reply by TobiMMXIX** [reply]
Unfortunately, I no longer have the Audible app installed.

**Reply by audiamus** [reply]
Legacy Audible Manager might also work. I think it's still available from CNet and may be on other sites, too. If you are able to activate it, it will store your activation bytes in the Windows registry and AAX Audio Converter will find them.

If that fails: re-download with Book Lib Connect.

**Reply by TobiMMXIX** [reply]
I didn't expect that the Legacy Audible Manager could still be activated, so I didn't look for it on other sites. It worked. Thank you very much.  👍
